corehq/apps/domain/templates/domain/manage_releases.html

```diff
@@ -0,0 +1,64 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% requirejs_main 'app_manager/js/manage_releases' %}
+
+{% block page_content %}
+    {% initial_page_data 'enabled_app_releases' enabled_app_releases %}
+    {% registerurl 'deactivate_release_restriction' domain '---'%}
+    {% registerurl 'activate_release_restriction' domain '---'%}
+    <div class="row">
+        <div class="col-sm-12">
+            <div class="panel panel-modern-gray panel-form-only">
+                <div class="panel-body">
+                    <form class="form-horizontal disable-on-submit" id="manage-app-releases" action=""
+                          method='post'>
+                        {% crispy manage_releases_form %}
+                    </form>
+                {% if enabled_app_releases %}
+                    <table id="managed-releases" class="table">
+                        <thead>
+                            <tr>
+                                <th>{% trans 'Application' %}</th>
```

Comment:
It's good practice to explicitly set column proportions by adding a `col-sm-X` class to each <th>. It prevents tables from getting wonky if someone has a really long application or location name.